### PR TITLE
chore: Portuguese translations for nutrients.txt

### PR DESCRIPTION
**Pull request by laralem**
Adding and fixing Portuguese translations for nutrients.txt

Added new nutrient:
en:Nitrate

### CODE REVIEW

File: taxonomies/nutrients.txt [hunk 1]
```diff
@@ -1469,7 +1470,7 @@ nb:fett
 nl:vet
 nn:feitt
 pl:tłuszcz
-pt:gordura
+pt:Gorduras/lípidos, Gordura, Lípidos
```
Comment (VaiTon) [inline taxonomies/nutrients.txt:1473]:
```suggestion
pt:Gorduras, Gordura, Lípidos
```

File: taxonomies/nutrients.txt [hunk 2]
```diff
@@ -1555,9 +1558,9 @@ nb:Mettet fett
 nl:Verzadigde vetzuren
 nn:metta feitt
 pl:Kwasy tłuszczowe nasycone, tłuszcz nasycony
-pt:Gorduras saturadas, gordura saturada
+pt:Gorduras/lípidos/ácidos gordos saturados, Gordura saturada, Ácidos gordos saturados, Lípidos saturados, dos quais saturados, Gorduras saturadas
```
Comment (VaiTon) [inline taxonomies/nutrients.txt:1561]:
```suggestion
pt:Gorduras saturados, Gordura saturada, Ácidos gordos saturados, Lípidos saturados, dos quais saturados, Gorduras saturadas
```

File: taxonomies/nutrients.txt [hunk 3]
```diff
@@ -2388,9 +2391,9 @@ nb:Enumettet fettsyre
 nl:Enkelvoudig onverzadigde vetzuren
 nn:einumetta feittsyre
 pl:Kwasy tłuszczowe jednonienasycone, jednonienasycony kwas tłuszczowy
-pt:Gorduras monoinsaturadas
+pt:Gorduras/límpidos monoinsaturados, Gordura monoinsaturada, Ácidos gordos monoinsaturados, Lípidos monoinsaturados, Gorduras monoinsaturadas
```
Comment (VaiTon) [inline taxonomies/nutrients.txt:2394]:
```suggestion
pt:Gorduras monoinsaturados, Gordura monoinsaturada, Ácidos gordos monoinsaturados, Lípidos monoinsaturados, Gorduras monoinsaturadas
```

File: taxonomies/nutrients.txt [hunk 4]
```diff
@@ -3594,7 +3600,7 @@ or:ସୋଡ଼ିଅମ
 pa:ਸੋਡੀਅਮ
 pi:सोडियम
 pl:Sód
-pt:Sódio
+pt:Sódio, Na+, Na
```
Comment (VaiTon) [inline taxonomies/nutrients.txt:3603]: Is there any product with "Na+" specified as a nutrient?

File: taxonomies/nutrients.txt [hunk 5]
```diff
@@ -5793,7 +5799,7 @@ pi:क्याल्सियम
 pl:Wapń
 # pms:Càuss
 # pnb:کیلشیم
-pt:Cálcio
+pt:Cálcio, Ca2+, Ca
```
Comment (VaiTon) [inline taxonomies/nutrients.txt:5802]: Same as before, is there any product specifying "Ca2+" or "Ca" as a nutrient?

File: taxonomies/nutrients.txt [hunk 6]
```diff
@@ -5653,7 +5659,7 @@ pa:ਸਿਲੀਕਾਨ ਡਾਈਆਕਸਾਈਡ
 pl:ditlenek krzemu
 # pnb:سلیکان ڈائیآکسائیڈ
 ps:سليكان ډای اکسايډ
-pt:dióxido de silício
+pt:Sílica, Dióxido de silício, SiO2
```
Comment (VaiTon) [inline taxonomies/nutrients.txt:5662]: Same as before, is there any poduct specifying "SiO2" as a nutrient?

**Review by VaiTon — CHANGES_REQUESTED** [review]
Thank you for your PR!

I'm not sure about adding the chemical symbols (Mg, Na, ecc.). AFAIK they are not used in nutrition tables.

File: taxonomies/nutrients.txt [hunk 7]
```diff
@@ -673,9 +674,9 @@ or:ଶ୍ଵେତସାର
 pa:ਕਾਰਬੋਹਾਈਡਰੇਟ
 pl:Węglowodany
 # pnb:کاربوہائیڈریٹ
-pt:Hidratos de carbono
+pt:Carboidratos/hidratos de carbono, Glicídios, Glícidos, Glucídios, Glúcides, Hidratos de carbono
```
[the remark this answers is not in the document]
Comment (laralem) [reply taxonomies/nutrients.txt:677]: In Portugal, chemical symbols are shown only in water bottles (not all brands show only chemical symbols, only a few, may be 50% or 20% maximum of them, I'm guessing). For example:
https://pt.openfoodfacts.org/produto/56004083/pedras-salgadas-agua-mineral-natural-gasocarbonica 
nutrition_pt 43 full

To the common people, its a challenge to enter that information in OpenFoofFacts without knowing what they mean.

File: taxonomies/nutrients.txt (hunk 4 above)
Comment (laralem) [reply taxonomies/nutrients.txt:3603]: Is shown in this label in the same water bottle:
nutrition_pt 43 full_2

File: taxonomies/nutrients.txt (hunk 5 above)
Comment (laralem) [reply taxonomies/nutrients.txt:5802]: Please see the previous image.

File: taxonomies/nutrients.txt (hunk 6 above)
Comment (laralem) [reply taxonomies/nutrients.txt:5662]: Please see the previous image.

File: taxonomies/nutrients.txt (hunk 1 above)
Comment (laralem) [reply taxonomies/nutrients.txt:1473]: These suggestions by me where to solve a minor/big problem: OpenFoodFacts at this moment only have 1 Portuguese interface translation. European Portuguese and Brazilian Portuguese are not very different, but sometimes the words are not the same. In Brazil "Fats" are shown as "Gorduras" in nutrition labels and in Portugal "Lípidos" (don't know in other Portuguese speaking countries, generally they use European Portuguese). So translating "Fats" as "Gorduras/lípidos" solves this problem.

I'm guessing this translation is shown in OpenFoodFacts while editing nutrition facts in a product, right? And only the first word "Gorduras" in "pt:Gorduras, Gordura, Lípidos" is shown, the others "Gordura, Lípidos" are shown if we search/add another nutrient. I'm I wrong?

File: taxonomies/nutrients.txt (hunk 7 above)
[the remark this answers is not in the document]
Comment (laralem) [reply taxonomies/nutrients.txt:677]: Found another one
https://pt.openfoodfacts.org/produto/5604868001039/
nutrition_pt 25 400

**Comment by aleene** [comment]
I wonder where the file nutrients.txt is used. I think only as suggestions for the nutritional table and as translations. Adding synonyms here has no impact. So better think of the best way to present the nutrition, especially so that its understood by Portuguese speakers in Portugal and Brazil.

@stephanegigandet ?

File: taxonomies/nutrients.txt (hunk 1 above)
Comment (alexgarel) [reply taxonomies/nutrients.txt:1473]: you are right @laralem

Comment (alexgarel) [reply taxonomies/nutrients.txt:1473]: But I'm not sure that's a good way to deal with it. @stephanegigandet what is your opinion ?

Comment (stephanegigandet) [reply taxonomies/nutrients.txt:1473]: sounds good to me until we have a better way to deal with language variants

**Review by stephanegigandet — APPROVED** [review]

File: taxonomies/nutrients.txt (hunk 3 above)
Comment (alexgarel) [reply taxonomies/nutrients.txt:2394]: See above, we consider this ok.

**Review by alexgarel — COMMENTED** [review]
Hi @laralem 

Really sorry to have this branch stalled for so long…

Would you have the courage to report your changes on a new branch (so that we avoid conflicts that have accumulated over time) ?

Thanks a lot for your contribution.

**Comment by benbenben2** [comment]
should we close this PR, since it has been handled by https://github.com/openfoodfacts/openfoodfacts-server/pull/8312
?

**Comment by stephanegigandet** [comment]
Closing as this PR was included in https://github.com/openfoodfacts/openfoodfacts-server/pull/8312 by @benbenben2 
Thank you @laralem and @benbenben2 !

**Pull request closed without merging**